packages/convex-helpers/package.json

```diff
@@ -24,6 +24,18 @@
       "types": "./dist/react/sessions.d.ts",
       "default": "./dist/react/sessions.js"
     },
+    "./react/cache/core": {
+      "types": "./dist/react/cache/core.d.ts",
+      "default": "./dist/react/cache/core.js"
+    },
+    "./react/cache/hooks": {
+      "types": "./dist/react/cache/hooks.d.ts",
+      "default": "./dist/react/cache/hooks.js"
+    },
+    "./react/cache/provider": {
+      "types": "./dist/react/cache/provider.d.ts",
+      "default": "./dist/react/cache/provider.js"
+    },
```

Comment:
Is it a React requirement that these be 3 separate files? I know the organization is nice, but I'd prefer to have it all in `react/cache.ts` so it's one entrypoint per feature

Reply:
Yeah. SSR messes with this. "use client" (hook) code and provider code can't be in the same file.

Reply:
... but the hooks file doesn't have "use client" at the top? At the least I'm taking out core, since none of the code is shared between them